### PR TITLE
Migrate scalability release branch jobs to pod-utils

### PR DESCRIPTION
**Pull request by tosi3k**

### CODE REVIEW

**Comment by tosi3k** [comment]
Hold until Monday.

/hold

**Comment by tosi3k** [comment]
Master branch presubmits were successfully migrated to pod-utils - some of the PRs that were using a new version of `pull-kubernetes-e2e-gce-100-performance` got merged already: https://prow.k8s.io/job-history/gs/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-e2e-gce-100-performance.

/assign @mborsz

File: config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml [hunk 1]
```diff
@@ -1451,7 +1491,7 @@ presubmits:
           value: "true"
         - name: BUILD_TYPE
           value: bazel
-        image: gcr.io/k8s-testimages/krte:v20210312-67f589a-1.18
+        image: gcr.io/k8s-testimages/krte:v20210311-6badf7a-1.18
```
Comment (mborsz) [inline config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml:1494]: ??

File: config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml [hunk 2]
```diff
@@ -695,7 +717,7 @@ periodics:
         value: "true"
       - name: BUILD_TYPE
         value: bazel
-      image: gcr.io/k8s-testimages/krte:v20210312-67f589a-1.19
+      image: gcr.io/k8s-testimages/krte:v20210311-6badf7a-1.19
```
Comment (mborsz) [inline config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml:720]: Unintended change?

Comment (tosi3k) [reply config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml:720]: > Unintended change?

Unskillful rebase attempt ;) thanks for catching this. Let me fix that in a minute.

Comment (tosi3k) [reply config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml]: Fixed. Thanks again for catching this!

**Pull request merged**